src/components/reset-password-2fa-auth-app/reset-password-2fa-auth-app-controller.ts

```diff
@@ -23,7 +23,7 @@ import { isLocked, timestampNMinutesFromNow } from "../../utils/lock-helper";
 const TEMPLATE_NAME = "reset-password-2fa-auth-app/index.njk";
 export function resetPassword2FAAuthAppGet(): ExpressRouteFunc {
   return async function (req: Request, res: Response) {
-    if (isLocked(req.session.user.wrongCodeEnteredLock)) {
+    if (isLocked(req.session.user.wrongCodeEnteredPasswordResetMfaLock)) {
```

Comment:
We're not actually setting this lock anywhere, so this will never currently be locked.

Reply:
Oh sorry, I see it in the previous commit. I'd probably move adding it to the types in the previous commit too, since that commit wouldn't make sense without it?